3rdparty/dxc/CMakeLists.txt

```diff
@@ -60,11 +60,29 @@ else()
 endif()
 
 set(DXC_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/build" CACHE INTERNAL "")
-execute_process(COMMAND "${CMAKE_COMMAND}" -S "${CMAKE_CURRENT_SOURCE_DIR}/dxc" -B "${DXC_BUILD_DIR}" -G "${CMAKE_GENERATOR}" "-Ax64" -T "${CMAKE_GENERATOR_TOOLSET}" ${NBL_DXC_CMAKE_OPTIONS}
+# execute_process(COMMAND "${CMAKE_COMMAND}" -C "${CMAKE_CURRENT_SOURCE_DIR}/dxc/cmake/caches/PredefinedParams.cmake" -S "${CMAKE_CURRENT_SOURCE_DIR}/dxc" -B "${DXC_BUILD_DIR}"-D CMAKE_BUILD_TYPE:STRING=Debug   ${NBL_DXC_CMAKE_OPTIONS}
+# 	RESULT_VARIABLE DXC_CMAKE_RESULT
+# 	OUTPUT_VARIABLE DXC_CMAKE_STREAM_PIPE
+# )
+
+execute_process(COMMAND "${CMAKE_COMMAND}" 
+	-S "${CMAKE_CURRENT_SOURCE_DIR}/dxc"
+	-C "${CMAKE_CURRENT_SOURCE_DIR}/dxc/cmake/caches/PredefinedParams.cmake"
+	-D "CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE" 
+	-D "CMAKE_BUILD_TYPE:STRING=Debug" 
+	-D "CMAKE_C_COMPILER:FILEPATH=/usr/bin/gcc" 
+	-D "CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/g++"
+	-B "${DXC_BUILD_DIR}" 
+	-G "Ninja"
 	RESULT_VARIABLE DXC_CMAKE_RESULT
 	OUTPUT_VARIABLE DXC_CMAKE_STREAM_PIPE
 )
 
+# execute_process(COMMAND "${CMAKE_COMMAND}" -C "${CMAKE_CURRENT_SOURCE_DIR}/dxc/cmake/caches/PredefinedParams.cmake" -S "${CMAKE_CURRENT_SOURCE_DIR}/dxc" -B "${DXC_BUILD_DIR}" -G "${CMAKE_GENERATOR}" "-Ax64" -T "${CMAKE_GENERATOR_TOOLSET}" ${NBL_DXC_CMAKE_OPTIONS}
+# 	RESULT_VARIABLE DXC_CMAKE_RESULT
+# 	OUTPUT_VARIABLE DXC_CMAKE_STREAM_PIPE
+# )
```

Comment:
@pollend you compile with GCC because Clang doesn't work?

Reply:
nope I just forgot to undo it.